rules/S6496/cfamily/metadata.json

```
@@ -0,0 +1,18 @@
+{
+  "title": "Pass small, trivially-copyable objects by value",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "2min"
+  },
+  "tags": [
+  ],
+  "defaultSeverity": "Minor",
+  "ruleSpecification": "RSPEC-6496",
+  "sqKey": "S6496",
+  "scope": "All",
+  "defaultQualityProfiles": [
+  ],
+  "quickfix": "covered"
```

```suggestion
  "quickfix": "targeted"
```
Am I missing something?